pkg/littlefs/Makefile.include

```diff
@@ -3,3 +3,8 @@ INCLUDES += -I$(PKGDIRBASE)/littlefs
 ifneq (,$(filter littlefs_fs,$(USEMODULE)))
   DIRS += $(RIOTBASE)/pkg/littlefs/fs
 endif
+
+# Set vfs file and dir buffer sizes
+CFLAGS += -DVFS_FILE_BUFFER_SIZE=56 -DVFS_DIR_BUFFER_SIZE=44
+# Reduce LFS_NAME_MAX to 31 (as VFS_NAME_MAX default)
+CFLAGS += -DLFS_NAME_MAX=31
```

Comment:
Shouldn't it be set to `VFS_NAME_MAX` directly?

Reply:
`littlefs2/lfs.h` doesn't include `vfs.h` - we could patch it though.

Reply:
IIRC, there is a mechanism in littlefs to include a config file. I guess it could be used to include `vfs.h`.

Reply:
I've tried that, but it seems like a patch to littlefs itself is necessary for the config file to be included everywhere.
I'd like to keep that to a separate PR.